app/_includes/head.html

```diff
@@ -26,7 +26,8 @@
   <meta name="google-site-verification" content="CrU3zp02dNKTe8NSAipL4NCPkrIjDXG8fViTZ-MIzP4"/>
 
   <!-- New Relic -->
-  <script src="/assets/newrelic.js"></script>
```

Comment:
Seems like this should have been a separate commit. Any reason why it wasn't loaded though? Just curious!